colossus/src/main/scala/colossus/service/ServiceServer.scala

```diff
@@ -110,7 +112,7 @@ extends Controller[I,O](codec, ControllerConfig(50)) {
       requests.hit(tags = tags)
       latency.add(tags = tags, value = (System.currentTimeMillis - done.creationTime).toInt)
       concurrentRequests.decrement()
-      push(comp) {
+      push(comp, done.creationTime) {
```

Comment:
Blegh..annoying that I just can't comment on unaffected lines in a PR.  This comment is not directed to this line.  It is for the comment above this function which is slightly out of date.  The comment states: 

/**
   * This is the only function that actually writes to the channel.  It will
   * write any queued responses until it hits an incomplete promise (or the
   * write buffer fills up)
   */
Which isn't wholly true anymore..this function actually will just enqueue the finished requests/responses.  When they get written is up to the controller